materials.lua

```diff
@@ -0,0 +1,363 @@
+local nici = 1
+-- Default Materials
+
+--default:stone
+mymillwork.register("default:stone",
+    "default_stone",
```

Comment:
I'm not the mod owner, but I think following points might improve the code:
1) Tabs instead of spaces, as seen in `register.lua` and `millwork.lua`
2) Automating this registration list: (untested)
```Lua
local nodes = {
	"default:stone", "default:cobble", "default:stonebrick"
}
for i, node_name in ipairs(nodes) do
	local def = minetest.registered_nodes[node_name]
	local model = node_name:gsub(":", "_")
	-- handle "default:stonebrick" and "default:desert_sandstone_brick"
	model = model:gsub("([^_]*)brick", "%1brick")

	local groups = table.copy(def.groups)
	groups.not_in_creative_inventory = nici
	mymillwork.register(node_name, model, def.description, def.tiles[1], groups)
end
```